cmd/backup-operator/README.md

```diff
@@ -0,0 +1,104 @@
+# etcd backup operator
+
+## Overview
+
+etcd backup operator backups the data of a etcd cluster running on [Kubernetes][Kube] to a remote storage such as AWS [S3][s3]. Currently, backup operator only supports saving backup to AWS S3.
+
+## Requirements
```

Comment:
Let's just get rid of `Requirements` section if it's the same as the top README.
We will have `Requirements` in top README and then each sub-doc have its additional requirements.

Reply:
alright.